ios/utils/RNSUIBarButtonItem.h

```diff
@@ -1,3 +1,5 @@
+#import <UIKit/UIKit.h>
```

Comment:
This was erroring out when building for visionOS, not sure why but I think it's good to have the import here